safehttp/incoming_request.go

```diff
@@ -41,6 +41,21 @@ func newIncomingRequest(req *http.Request) *IncomingRequest {
 	}
 }
 
+// GetMethod TODO
+func (r *IncomingRequest) GetMethod() string {
```

Comment:
Drop the "Get" prefix, here and everywhere else: https://golang.org/doc/effective_go.html#Getters.